### PR TITLE
[fr] Full 3.5 migration guide translation

### PR DESCRIPTION
**Pull request by HavokInspiration**
Full translation of the 3.5 migration guide.
I voluntarily left the references to next section out for the moment as they are not translated yet.
I'll update the migration guide as I translate the rest of the 3.next doc.

### CODE REVIEW

File: fr/appendices/3-5-migration-guide.rst [hunk 1]
```diff
@@ -4,20 +4,243 @@
...
+  le middleware CSRF à la place.
+* ``Cake\Datasource\TableSchemaInterface`` est dépréciée. Utilisez
+  ``Cake\Database\TableSchemaAwareInterface`` à la place.
+* ``Cake\Console\ShellDispatcher`` est dépréciée. Vous devez mettre à jour vos
+  Applications pour qu'elles utilisent ``Cake\Console\CommandRunner`` à la place.
+
+Dépréciation des Méthodes Get / Set combinées
+---------------------------------------------
+
+Dans le passé, CakePHP a utilisé des méthodes 'modal' qui proposait à la fois
```
Comment (antograssiot) [inline fr/appendices/3-5-migration-guide.rst]: proposaient

File: fr/appendices/3-5-migration-guide.rst [hunk 2]
```diff
@@ -4,20 +4,243 @@
...
+un mode get et un mode set. Ces méthodes compliquent l'auto-complétion des IDE
+et notre capacité à ajouter des *return type* stricts dans le futur. Pour ces
+raisons, ces méthodes get / set combinés sont scindées en méthodes get et set
```
Comment (antograssiot) [inline fr/appendices/3-5-migration-guide.rst]: combinées

File: fr/appendices/3-5-migration-guide.rst [hunk 3]
```diff
@@ -4,20 +4,243 @@
...
+  adapter HTTP personnalisés dans leurs clients.
+* Les sous-commandes composées de plusieurs mots devaient impérativement être
+  appelées avec leur nom camelBacked pour être utilisées. Les sous-commandes
+  peuvent maintenant être appelées via leur nom au format *underscored_names*.
+  Par exemple : ``cake tool initMyDb`` peut maintenant être appelée via
+  ``cake tool init_my_db``. Si vos Shells liaient 2 sous-commandes avec le même
+  nom mais 2 inflections différentes, seule la dernière commande liée fonctionnera.
+* ``SecurityComponent`` bloquera les requête POST qui ne passent pas de données
+  dans la requête (pas de *request data*). Cela aide à la protection des actions
+  qui créent des enregistrement en base en utilisant seulement les *defaults* de
```
Comment (antograssiot) [inline fr/appendices/3-5-migration-guide.rst]: Enregistrements

**Comment by HavokInspiration** [comment]
@antograssiot I took care of your comments.

**Comment by antograssiot** [comment]
Thanks :+1:

**Pull request merged**